tests/UnifiedSpecTests/EventCollector.php

```diff
@@ -120,7 +120,7 @@ private function handleCommandMonitoringEvent($event): void
             'name' => self::getEventName($event),
             'observedAt' => microtime(true),
             'commandName' => $event->getCommandName(),
-            'connectionId' => self::getConnectionId($event),
+            'connectionId' => $event->getServerConnectionId(),
```

Comment:
This logs and the `$eventList` property are never used. They can be useful for debugging.

Reply:
See: https://github.com/mongodb/mongo-php-library/pull/1428